examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+meanio-controller                      meanio                  shaiweinstein/k8s:v1                name=meanio                  1
+meanio-mongodb-controller              meanio-mongodb          mongo:2.6.8                         name=meanio-mongodb          1
+
+```
+
+Once that's up (it may take a few minutes to create the pod) you can list the pods, to verify that the mongodb and meanio are running:
+
+```shell
+$ kubectl get pods
+POD                                          IP                  CONTAINER(S)            IMAGE(S)                                   HOST                                                              LABELS                                                     STATUS
+meanio-controller-zwk1b                      10.244.0.13         meanio                  shaiweinstein/k8s:v1                kubernetes-minion-yxea.c.mean-io.internal/104.155.36.70   app=meanio,name=meanio,uses=meanio-mongodb                                  Running             23 hours
+meanio-mongodb-controller-nyxxv              10.244.0.11         meanio-mongodb          mongo:2.6.8                         kubernetes-minion-yxea.c.mean-io.internal/104.155.36.70   app=mongodb,name=meanio-mongodb                                             Running             23 hours
+
+```
+
+You will see a mongodb server pod, and meanio pod.
+
+### Step Four: Create the meanio service.
+
+Just like the others, you want a service to group your meanio pods (in case of running more than one relica)..
+The service is described in the file `examples/meanio/meanio-service.json`:
+
+```js
+{
+  "id": "meanio",
+  "kind": "Service",
+  "apiVersion": "v1beta1",
+  "port": 80,
+  "containerPort": "meanio-server",
+  "selector": {
+    "name": "meanio"
+  },
+  "labels": {
+    "name": "meanio"
+  },
+  "createExternalLoadBalancer": true
+}
+```
+
+If running a single node local setup, or single VM, you don't need `createExternalLoadBalancer`.
+
+```shell
+$ kubectl create -f examples/meanio/meanio-service.json
+meanio
+
+
+$ kubectl get services
+NAME                    LABELS                                    SELECTOR                     IP                  PORT
+meanio                  name=meanio                                                     name=meanio                  10.0.29.25          80
+meanio-mongodb          name=meanio-mongodb                                             name=meanio-mongodb          10.0.222.168        27017
+```
+
+### A few Google Container Engine specifics for playing around with the services.
+
+In GCE, `cluster/kubectl.sh` automatically creates forwarding rule for services with `createExternalLoadBalancer`.
+
+```shell
+$ gcloud compute forwarding-rules list
+NAME                  REGION      IP_ADDRESS     IP_PROTOCOL TARGET
+kubernetes-meanio europe-west1 130.211.xxx.xxx TCP         europe-west1/targetPools/kubernetes-meanio
```

Comment:
Is this IP address always guaranteed to start with `130.211`? If not, it might be more clear to just have `xxx.yyy.zzz.www` here.